src/audero-sticky.js

```diff
@@ -481,7 +475,8 @@ export
          placeholder.style,
          {
             visibility: 'hidden',
-            zIndex: getZIndex(this.element, this.settings.selector)
+            zIndex: getZIndex(this.element, this.settings.selector),
+            display: 'block'
```

Comment:
I think this is separate concern and @AurelioDeRosa should first decide which way to go #25.